### ISSUE TITLE
Autoprefixer reports false warnings.

### ISSUE THREAD

**Issue by TrevorKarjanis** [postcss/autoprefixer#1251]
Autoprefixer will report false warnings if the value of a declaration contains a CSS variable that contains the substring being tested against. For example, this declaration results in the following warning.
```css
width: calc(100% - var(--jqx-circular-progress-bar-fill-size));
```
```bash
WARNING in ./src/styles/styles.scss (./node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/raw-css-loader.js!./node_modules/postcss-loader/src??embedded!./node_modules/sass-loader/lib/loader.js??ref--13-3!./src/styles/styles.scss)
Module Warning (from ./node_modules/postcss-loader/src/index.js):
Warning

(200:5) Replace fill to stretch, because spec had been changed
```

Unfortunately, this is defined in a third-party library and occurs multiple times in each build run. It seems to occur because the processor is using indexOf.
```javascript
// lib/processor.js line 306
} else if (decl.value.indexOf('fill') !== -1) {
  result.warn(
    'Replace fill to stretch, because spec had been changed',
    { node: decl }
  )
}
```
autoprefixer@9.6.0
postcss@7.0.17

**Comment by ai** [comment]
Fixed 414efbe

**Comment by ai** [comment]
Released in 9.6.2
